DOCS-1827: Adds new second chance info to the rest of DOCS (#1035)
MultiSafepay/docs: content/tools/second-chance/how-does-it-work.md

meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
read_more: '.'
---

> When a customer initiates a payment, but does not finish it for any reason, a Second Chance reminder email can be sent. Included in these emails, there is a payment link that will allow the customer to finalize the payment. The first Second Chance email can be sent after **1 hour** and the second after **24 hours** after the initial transaction was started.

Reviewer: ```suggestion
When a customer initiates a payment but does not complete it, a Second Chance reminder email can be sent. Included in these emails, there is a payment link that will allow the customer to finalize the payment. The first Second Chance email can be sent after **1 hour** and the second after **24 hours** after the initial transaction was started.
```

* [Direct Debit](/payment-methods/banks/sepa-direct-debit/)

## Altering the Second Chance configuration
At the moment you can not change the Second Chance email in the settings. __As per GDPR requirements MultiSafepay does not submit emails by default. Please ensure that when enabling this option, there is a documented consent from the receiver for submitting an email related to the respective payment link.__ When enabled, the Second Chance will be sent twice within 24 hours and this cannot be adjusted (after 1 hour and after 24 hours).

For more information about how the Second Chance reminder can be sent through our API, please see the section on [generating a payment link](https://docs.multisafepay.com/api/#generating-a-payment-link)

> The Second Chance email can be personalised through an email template. More information on how to personalize a Second Chance email is available in [Personalize the Second Chance email](/tools/second-chance/how-to-personalize-the-second-chance-e-mail)

Reviewer: ```suggestion
The Second Chance email can be personalized through an email template. More information on how to personalize a Second Chance email is available in [Personalize the Second Chance email](/tools/second-chance/how-to-personalize-the-second-chance-e-mail)
```
